packages/odyssey-react/src/components/Select/Select.test.tsx

```diff
@@ -28,6 +34,27 @@ const tree = (props: Record<string, unknown> = {}) => (
   </Select>
 );
 
+const RefTree = (props: Record<string, unknown> = {}) => {
+  const selectRef = useRef<ChoicesHTMLSelectElement>(null);
+
+  useEffect(() => {
+    if (selectRef?.current?.choices) {
+      selectRef?.current?.choices.setChoices([
+        {
+          value: "ref",
+          label: "Choices!",
+        },
+      ]);
+    }
+  }, [selectRef]);
+
+  return (
+    <Select ref={selectRef} {...props} label={label} name={name}>
+      <Select.Option children={"No choices"} />
+    </Select>
+  );
+};
+
```

Comment:
consider moving this into the test body itself `it('...', () => {})` if it isn't shared across other tests rather than defining it here and accessing below via closure